source/common/runtime/runtime_features.cc

```diff
@@ -81,6 +81,7 @@ constexpr const char* runtime_features[] = {
 constexpr const char* disabled_runtime_features[] = {
     // Sentinel and test flag.
     "envoy.reloadable_features.test_feature_false",
+    "envoy.reloadable_features.alternative_least_request_weights",
```

Comment:
If we end up doing a real PR for this, it should be a real config option vs. one of these. I would just use a RuntimeFeatureFlag inside the LB config proto.